compiler/compile/VirtualGuard.cpp

```diff
@@ -133,10 +133,18 @@ TR::Node*
 TR_VirtualGuard::createVftGuard
 (TR_VirtualGuardKind kind, TR::Compilation * comp, int16_t calleeIndex,
  TR::Node* callNode, TR::TreeTop * destination, TR_OpaqueClassBlock *thisClass)
+   {
+   return createVftGuardWithReceiver(kind, comp, calleeIndex, callNode, destination, thisClass, callNode->getSecondChild());
```

Comment:
Shouldn't this use the Node api to get the index of the child to read rather than hardcoding it?

Reply:
@andrewcraik Fixed.